spec/Section 1 -- Overview.md

```diff
@@ -18,7 +18,10 @@ The GraphQL Composite Schemas specification has a number of design principles:
   responsible for serving within the context of the larger schema, referencing
   and extending that which is provided by other source schemas. The GraphQL
   Composite Schemas specification does not describe how to combine arbitrary
-  schemas.
+  schemas.  
+  Note: Tooling may be built to transform existing or external schemas into
```

Comment:
spec-md does not support double-space forced newlines because Lee doesn't like that that's a feature of Markdown.

Also, I'm not sure this will work correctly as a "Note" callout under spec-md in this position? I'd have to render it with spec-md to be sure, but I think it would be better somewhere after the bulleted list?

Reply:
It renders like this:

![image](https://github.com/user-attachments/assets/984ea5d5-7621-4b9c-80e5-437dc10228a4)

If you add an empty line before the note, you get:

![image](https://github.com/user-attachments/assets/e5b68eba-d6a3-44e7-9cdf-f11f13b3d423)

Better, but ideally it would be indented.